mmv1/products/cloudrun/Service.yaml

```diff
@@ -875,6 +875,28 @@ properties:
                             Driver-specific attributes. The following options are supported for available drivers:
                               * gcsfuse.run.googleapis.com
                                 * bucketName: The name of the Cloud Storage Bucket that backs this volume. The Cloud Run Service identity must have access to this bucket.
+                    - !ruby/object:Api::Type::NestedObject
+                      name: nfs
```

Comment:
We'll want to add a test that verifies that this works properly in the provider, a test configuration can be added [here](https://github.com/GoogleCloudPlatform/magic-modules/tree/77041bfd26766336f1c2a47818425c46712e5b7a/mmv1/templates/terraform/examples) as a as a `cloud_run_service_nfs.tf.erb` file.

Reply:
I've been trying and I'm not sure it's possible to make a test for this. The test requires a VPC, and tearing down the VPC keeps failing because resource locks aren't immediately released when the Cloud Run Service and/or VPC Connector are deleted.